katas/content/nonlocal_games/ghz_classical_strategy/index.md

```diff
@@ -0,0 +1,11 @@
+In this task you have to implement three functions, one for each player's classical strategy.
+Note that they are covered by one test, so you have to implement all of them to pass the test.
+For non-deterministic strategies, DrawRandomBoolean or [DrawRandomInt](https://learn.microsoft.com/qsharp/api/qsharp-lang/microsoft.quantum.random/drawrandomint) functions from Q# library can be used.
+
+**Inputs:**
```

Comment:
This formulation is a bit confusing, since we used a very similar one in the previous task to describe win condition inputs, all bits given to one operation. I'd say something like "In each function, the input is the starting bit of the corresponding player, and it should return the output bit chosen by that player."

Reply:
Done